src/main/resources/assets/traverse/lang/zh_cn.json

```
@@ -3,20 +3,20 @@
 	"biome.traverse.autumnal_wooded_hills":"秋木山脉",
 	"biome.traverse.autumnal_woods":"秋木树林",
 	"biome.traverse.cliffs":"悬崖",
-	"biome.traverse.coniferous_forest":"针叶森林",
-	"biome.traverse.coniferous_woodland_hills_biome":"针叶林山",
+	"biome.traverse.coniferous_forest":"冷杉森林",
+	"biome.traverse.coniferous_woodland_hills_biome":"冷杉林山",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
	"biome.traverse.coniferous_woodland_hills_biome":"冷杉林丘陵",
```